api-reference/10 UI Components/GridBase/1 Configuration/columnChooser/position.md

```diff
@@ -0,0 +1,157 @@
+---
+id: GridBase.Options.columnChooser.position
+type: PositionConfig
+default: undefined
+---
+---
+##### shortDescription
+Configures the column chooser's position.
+
+---
+You can specify the [my](/Documentation/ApiReference/Common/Object_Structures/PositionConfig/my/), [at](/Documentation/ApiReference/Common/Object_Structures/PositionConfig/at/), and [of](/Documentation/ApiReference/Common/Object_Structures/PositionConfig/#of) properties to position a column chooser. 
+
+---
+##### jQuery
+
+    <!--JavaScript-->
+    $(function(){
+        $("#{widgetName}").dx{WidgetName}({
+            // ...
+            columnChooser: {
+                // ...
+                position: {
```

Comment:
The snippet is correct, but the example is confusing. I would suggest using this:

`
position: {
  my: "right top",
  at: "right top",
  of: '.dx-datagrid'
}
`